isolate_example/lib/page_one.dart

```diff
@@ -12,20 +12,115 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 
-class PerformancePage extends StatelessWidget {
+class PerformancePage extends StatefulWidget {
+  @override
+  _PerformancePageState createState() => _PerformancePageState();
+}
+
+class _PerformancePageState extends State<PerformancePage> {
+  Future<void> computeFuture = Future.value();
+
   @override
   Widget build(BuildContext context) {
     return Center(
       child: Column(
         mainAxisSize: MainAxisSize.min,
         children: <Widget>[
           SmoothAnimationWidget(),
+          Container(
+            alignment: Alignment.bottomCenter,
+            padding: EdgeInsets.only(top: 150.0),
+            child: Column(
+              children: <Widget>[
+                FutureBuilder<void>(
+                  future: computeFuture,
+                  builder: (BuildContext context, AsyncSnapshot snapshot) {
```

Comment:
Generally I prefer to leave off the type names for lambdas like these. If you feel strongly about it, you're free to ignore me, but I think you've done a good enough job with the param names to get their types across.